src/test/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/integration/GoalControllerTest.kt

```diff
@@ -361,6 +364,30 @@ class GoalControllerTest : IntegrationTestBase() {
           .returnResult().responseBody
 
       assertThat(goalEntity?.title).isEqualTo("New Goal Title")
+      assertThat(goalEntity?.targetDate).isNull()
+      assertThat(goalEntity?.status).isEqualTo(GoalStatus.FUTURE)
+    }
+
+    @Test
+    fun `should update an active goal into a future goal`() {
+      val goalRequestBody = Goal(
+        targetDate = null,
+        status = GoalStatus.FUTURE,
+      )
+
+      val goalUuid = "379f986a-c4f8-4c27-bdda-2ccb0aebb6a6"
+
+      val goalEntity: GoalEntity? =
+        webTestClient.patch().uri("/goals/$goalUuid").header("Content-Type", "application/json")
+          .headers(setAuthorisation(user = authUuid + "|Tom C", roles = listOf("ROLE_RISK_INTEGRATIONS_RO")))
```

Comment:
`"$authUuid|Tom C"` is more idiomatic.

Reply:
I'll refactor this across all the relevant controller tests to extract it into a class-level variable

Reply:
Fixed in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/84/commits/1c7e10887f31539fa5e42e1242af13e6911d67af